app/dashboard/views.py

```diff
@@ -300,6 +301,42 @@ def new_interest(request, bounty_id):
     })
 
 
+@require_POST
+def postComment(request):
+    profile_id = request.user.profile if request.user.is_authenticated and hasattr(request.user, 'profile') else None
+    if profile_id is None:
+        return JsonResponse({
+            'success': False,
+            'msg': '',
+        })
+
+    sbid = request.POST.get('standard_bounties_id')
+    bountyObj = Bounty.objects.filter(standard_bounties_id=sbid).first()
+    fbAmount = FeedbackEntry.objects.filter(sender_profile=profile_id, feedbackType='approver', bounty=bountyObj).count()
+    if fbAmount > 0:
+        return JsonResponse({
+            'success': False,
+            'msg': 'There is already a approval comment',
+        })
+    feedback_dict = request.POST.get('review',{})
+    kwargs = {
+        'bounty': bountyObj,
+        'sender_profile': profile_id,
+        'receiver_profile': bountyObj.fulfillments.last().profile,
+        'rating': feedback_dict.get('rating', '-1'),
+        'comment': feedback_dict.get('comment', 'No comment.'),
```

Comment:
I worry about potential script injection issues here due to the untrusted data: [OWASP Wiki](https://www.owasp.org/index.php/Injection_Theory#Untrusted_Data). 

However, maybe santizing that is beyond the scope of V1 of this feature. Lets add a note to make sure we don't forget that this data is untrusted and unsanitized 👍